UI/HUD.cs

```diff
@@ -157,6 +161,15 @@ public void RemoveNickLabel(int Id)
 	}
 
 
+	public void ShowDamageIndicator(Vector2 ShotFirePosition) {
```

Comment:
Use a Vector3 (so that the axis names line up with their contents) and don't `Flattened` until it is being used